src/Components/SingleReview.jsx

```diff
@@ -15,10 +14,11 @@ export default function SingleReview() {
       getReviewById(review_id).then((review) => {
         setReview(review);
       });
-  }, [review_id]);
+  }, [review_id,  <AddComment />]);
```

Comment:
components aren't meant to be in a dependancy...

Reply:
done =)